packages/react-scripts/config/webpack.config.js

```diff
@@ -143,9 +143,9 @@ module.exports = function (webpackEnv) {
             config: false,
```

Comment:
We should also need to specify the path to the postcss:
```js
 {
        // Options for PostCSS as we reference these options twice
        // Adds vendor prefixing based on your specified browser support in
        // package.json
        loader: require.resolve('postcss-loader'),
        implementation: require.resolve("postcss"),
        options: {
          postcssOptions: {
```

Reply:
That's not necessary, `postcss-loader` declares `postcss` as a peer dependency so it will get it from the parent (`react-scripts`) on its own.

Reply:
The problem I'm having is that under node_modules is postcss v7, but plugins like postcss-flexbugs-fixes depend on postcss v8, which causes errors when compiling.

So I think it might be better to specify implementation.